e2e/bitrise.yml

```diff
@@ -74,7 +104,7 @@ workflows:
           title: Step Test
           inputs:
             - target: $TARGET
-            - platform: ios,android
+            - platform: $PLATFORM
```

Comment:
This was accidentally hardcoded to `ios,android` instead of using the `PLATFORM` env var.

Reply:
Can we add separate tests where `$PLATFORM` is only `android` and `ios`? We might have a bug in the step when the platform is not `ios,android`, but I'm not 100% yet, @bitce is waiting for response from the customer.

Reply:
We already have `test_ios_device_build` which only builds `ios`!